### ISSUE TITLE
Book 05: Sentence 027

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#5]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/f1cc0607-589e-4126-b517-6006eaafa6c8)

```
# sent_id = Book_05_-_The_Omens.txt:27
# text = No ioan, tetzaujtl catca, netetzaujloia, tetzammachoia: in jtoca ioaltepuztli.
# text[norm] = No ihuan, tetzahuitl catca, netetzahuiloya, tetzamachoya: in itoca yohualtepoztli.
# text[orig] = No·ioan·,·tetzaujtl·catca·,·ne¶tetzaujloia·,·tetzammachoia·:·inj¶toca·ioaltepuztli·.
# text[osp] = [...] quiere dezir: hacha nocturna, [...]
# references = A&D/165:2
1	No	no	ADV	_		5	_	_	Folio=4v|Paragraph=4|Line=1|Norm=No|Gloss=también|Analysed=Yes
2	ioan	huan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Poss	5	_	_	Folio=4v|Paragraph=4|Line=1|Norm=ihuan|Gloss=con|Analysed=Yes
3	,	,	PUNCT	_		4	punct	_	Folio=4v|Paragraph=4|Line=1|Norm=,|Analysed=Yes
4	tetzaujtl	tetzahuitl	NOUN	_	State=Abs	5	_	_	Folio=4v|Paragraph=4|Line=1|Norm=tetzahuitl|Analysed=Yes
5	catca	catqui	VERB	_	Subcat=Intr	0	root	_	Folio=4v|Paragraph=4|Line=1|Norm=catca
6	,	,	PUNCT	_		7	punct	_	Folio=4v|Paragraph=4|Line=1|Norm=,|Analysed=Yes
7	netetzaujloia	tetzahuia	VERB	_	_	4	_	_	Orig=ne¶·tetzaujloia|Folio=4v,4v|Paragraph=4,4|Line=1,2|Norm=netetzahuiloya
8	,	,	PUNCT	_		9	punct	_	Folio=4v|Paragraph=4|Line=2|Norm=,|Analysed=Yes
9	tetzammachoia	mati	VERB	_	Aspect=Imp|Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Tran|Tense=Past|VerbForm=Fin|Voice=Pass	7	__Folio=4v|Paragraph=4|Line=2|Norm=tetzamachoya|Incorp=tetza|Analysed=Yes
9.1	tetza	tetzahuitl	NOUN	_	_	_	_	_	Incorporated=Yes
10	:	:	PUNCT	_		12	punct	_	Folio=4v|Paragraph=4|Line=2|Norm=:|Analysed=Yes
11	in	in	DET|SCONJ	_	_	12	_	_	Orig=inj¶·toca|Folio=4v,4v|Paragraph=4,4|Line=2,3|Norm=in
12	jtoca	toca	NOUN	_	_	4	_	_	Orig=inj¶·toca|Folio=4v,4v|Paragraph=4,4|Line=2,3|Norm=itoca|Gloss=su.nombre
13	ioaltepuztli	yohualtepoztli	NOUN	_	State=Abs	12	_	_	Folio=4v|Paragraph=4|Line=3|Norm=yohualtepoztli|Split=yohual·tepoztli|Analysed=Yes
14	.	.	PUNCT	_		5	punct	_	Folio=4v|Paragraph=4|Line=3|Norm=.|Analysed=Yes
```

**Comment by Lguyogiro** [comment]
Here's how I have this sentence:

![imagen](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/72c2b19c-6f81-4e15-bdf5-2786645f8d3e)

**Comment by ftyers** [comment]
- What's the motivation for having *No* dependent of *ihuan* ?
- Why the nested coordination for *netetzahuiloya* and *tetzammachoya*
- Shouldn't parataxis go to the root?

**Comment by ftyers** [comment]
```
Book_05_-_The_Omens.txt:60	 No ihuan, netetzahuiloya in pinahuiztli: iuhquin tocatl tlatlauhqui, tlatlactic.	
Book_05_-_The_Omens.txt:27	 No ihuan, tetzahuitl catca, netetzahuiloya, tetzamachoya: in itoca yohualtepoztli.	
Book_05_-_The_Omens.txt:46	 No ihuan, intla ye aca mococoa: in ipan tzatzitiquiza, quihtoaya ca ayocmo quizaz: ca ticcahuazque.	
Book_05_-_The_Omens.txt:56	 No ihuan, netetzahuiloya in tochin: in ihcuac aca ichan calaquiya, quihtoa in millaca, in milpan tlacah: ye tlalpolihuiz in ichan.	
Book_05_-_The_Omens.txt:91	 No ihuan, netetzahuiloya moteittitiaya: in itoca cuitlapanton, anozo cintanaton: mihtoa in Acolhuacan centlapachton: iuh tocayotilo.	
Book_05_-_The_Omens.txt:14	 No ihuan, quitetzammatia: in ihcuac aca ipan huehuetzca, huactli: in iuh conihtoa, ome huitz quihtoa: in quenman cualli, quenman amo cualli.	
Book_05_-_The_Omens.txt:85	 No ihuan, tetzahuitl catca, tetetzahuiaya: in tlacanexquimilli, iuh mihtoa: zan ye mochi yeh inahual, inecuepaliz, inenextiliz in Tezcatlipoca: in ihcuac ittoya, zan momimilotiuh, quiquinacatiuh, tehtentiuh.	
Book_05_-_The_Omens.txt:41	 No ihuan netetzahuiloya, tlatetzahuiaya, tetzamachoya, in tecolotl choca: in ihcuac tla aca itlapantenco, anozo ixacalticpac, icuauhticpac chocatica: in quicaqui quihtotica: tecolo, o, o, tecolooo, ihuin in caquizti, in choca.	
Book_05_-_The_Omens.txt:44	 No ihuan netetzahuiloya, tetzauhittoya, in chicuatli: anozo chichtli in ihcuac aca, itlapanixcuac tzatzi: tzatzitiquiza, in quihtoa: cuel, ic quineltocaya, ca ye aca miquiz: ye cemeh miquizque in chaneque: ye cemeh cocolizcuizque.	
Book_05_-_The_Omens.txt:80	 No ihuan, netetzahuiloya in tlacahuiyac: in ihcuac yohualtica moteittitia, mihtoa: zan ye moche inahual, inecuepaliz in tlacatecolotl Tezcatlipoca, moch ic teca mocayahua: in ihcuac aca iuh quimottitia, ye quitetzahuia in azo ye yaomiquiz, azo ye huicoz.	
Book_05_-_The_Omens.txt:69	 No ihuan in epatl, netetzahuiloya: in ihcuac aca ichan calaqui, anozo oncan mopilhuatia: quihtoa, ye miquiz in chane: ipampa iuh quitoaya, ca amo inentlan, amo inemiyan in callan, in techan: ca milpan, zacatlan, memetlan nonopallan, in inemiyan, in iquizayan.	
Book_05_-_The_Omens.txt:52	 No ihuan netetzahuiloya in cozamatl, anozo cozatli: in ihcuac aca ichan calaquiya: anozo ohtlica, cana quinamiqui: quiyacahuiltequi, ixpan tlaxtlapaloa: quihtoaya, azo ye itla conmonamictiz tetolini: azo ye itla ipan oncholoz, amo huel yaz in campa yaz nequi: azo acame inmac huetzitiuh, quimictizque: anozo ye tlahtolli, itech onehuaz: azo ye aca, tlahtolli conehualtiz: contlahtollaquiliz, contlahtolchichihuiliz.	
Book_05_-_The_Omens.txt:74	 No ihuan, netetzahuiloya in azcame, in tlatlauhque, anozo tliltique: in ihcuac aca, ichan moloni, mopotza: quinextiaya in tecocoliliztli: quihtoaya in ihcuac oittoque, azo aca totecocolicauh itla otechtoquilico: iuh mihtoa, iuh nemacho in tlein quitetoquilia, tecocoliani: yehhuan in mocuepa azcame, ihcuac in ye huehcahua, tlein oquitocaque: mochi yehhuan intech tlami in tecocoliani, in temiquitlanini, in temiquiztemachique, in temiquiztemachiani, in tehueyiyecoani, in atlanca in tlatlacatecolo: in motenehua, tepoloani yehhua quintoca, quintlalia in azcame.	
```

**Comment by Lguyogiro** [comment]
I have come to think that the "in" here is actually behaving pronominally. Lockhart gives the example of "in quichihua amo qualli" "what he's doing is not good."

**Comment by ftyers** [comment]
Here is another suggestion, this time with *catca* as a copula:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/26971372-bef5-47b6-8ce5-72b3bcbac396)

Do you think that *in itoca yohualtepoztli* is modifying the incorporated object *tetza-* in *tetzammachoya* ? e.g. *su nombre [del agüero] es hacha nocturna* ?

**Comment by ftyers** [comment]
The other question is if `advcl` is the right relation for *in itoca yohualtepoztli*. Some more examples of *`VERB`: in itoca ...* for comparison:

- *No ihuan, netetzahuiloya moteittitiaya: in itoca cuitlapanton, anozo cintanaton: mihtoa in Acolhuacan centlapachton: iuh tocayotilo.*
- *Ic matlactli omome capítulo; oncan mihtoa: in occentlamantli tetzahuitl catca: in itoca catca: tlacanexquimilli.*
- *Nican tlami, nican tzonquiza, inic ilhuitl mochihuaya: in itoca Hueyi tecuilhuitl.*

**Comment by Lguyogiro** [comment]
> The other question is if advcl is the right relation for in itoca yohualtepoztli. Some more examples of VERB: in itoca ... for comparison:

if we give _in_ some pronominal properties, we could interpret that last clause as "that which is called yohualtepoztli", and make it a core argument of one of the preceding verbs _netetzaujloia_ or _tetzammachoya_

**Comment by ftyers** [comment]
So in #34 we have a *tetzahuitl catca* with a subject, and here there is one seemingly without a subject, could it be that *in itoca yohualtepoztli* is the subject?

**Comment by Lguyogiro** [comment]
That's what I was thinking. However, if we treat _in itoca yohualtepoztli_ as a copular nominal clause "its name is night-axe", then making it the subject seems strange. On the other hand if, as I mentioned in my previous comment, we understand this to mean "that which's name is night-axe", then I think making it the subject is nice and, dare I say, elegant.

**Comment by Lguyogiro** [comment]
this is maybe a little weird but

![image](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/bae59f3a-038a-4d94-841e-e214e7d39211)

(along the lines of "that which's name night-axe")

**Comment by ftyers** [comment]
> this is maybe a little weird but

Hmm, can we find more examples of *in itoca X* in a similar construction? It just seems odd to go with a whole new POS... 

* *Inic cempohualli on caxtolli omeyi capítulo: itechpa tlatoa injlhujtl, in mo teneoaia: O auh quil tamalqualiztli, in mochihuaya in ipan ic tlamatlacti in metztli in tlac pac omihto: auh inihcuac y, quil hui qujx tiliaia, in inteouh catca, in itoca Ixcocauhqui*
* *Ic matlactli omome capítulo; oncan mihtoa: in occentlamantli tetzahuitl catca: in itoca catca: tlacanexquimilli.*
* *Auh in tlacopa tlatoani, catca in itoca Chimalpopoca, ipan in mic ixiuhtzin totecuiyo Jesu Cristo 1489.*
* *Inic centlamantli: intoca tolteca, iuh mihtoa: yehhuantin achto nemjco, in nican tlalli ipan: in mihtoa mexicatlalpan, in chichimeca tlalpan: auh quezqujtzonxivitl nenque, in ompa tullantzinco, in ca nelli ompa nenque: miec innezca qujtlaliteoaque, in ompa quichiuhque in jntevpan catca, in itoca oapalcalli, in axcan caca ca mani: tel inic ahuel polihui, ca tetl ca texcalli: ...*
* *... huel tlatlamach tlalilli, huellatoltecahuilli, in jtlateumatian catca, in inteopixcauh catca, in itoca Quetzalcoatl, huellamahuicolli catca.*
* *Vellatevmatinj catca, ca za ce in inteouh qujcemmatia, in quinotzaya, in quitlatlauhtiaya, in itoca Quetzalcoatl, in jntlamacazcauh catca, in inteopixcauh: zan no itoca Quetzalcoatl: ...* 

> Lockhart has sentences like "in quichihua amo qualli" "What he is doing is not good." Although I believe this type of construction uses ccomp, so that brings me back to my initial analysis of this clause:

I like this more, but how does it relate to the main clause...

**Comment by Lguyogiro** [comment]
I guess if they all share the same subject, which we could say is the last clause ("that which is called night axe"), then conj for the other clauses would make sense.

**Comment by ftyers** [comment]
> > The other question is if advcl is the right relation for in itoca yohualtepoztli. Some more examples of VERB: in itoca ... for comparison:
> 
> I lean towards `parataxis:shared` here. The subject of _catca_, _netetzaujloia_, and _tetzammachoya_ is the same thing whose name is "night axe".
> 
> Alternatively, a suggestion you will love: `conj`:
> 
> "it was a portent, (and) it was considered portentous, (and) it is known as a portent, (and) its name is night axe".

So like this?

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/2a3cb659-cf21-420e-acaf-9df0c518d6a4)

**Comment by Lguyogiro** [comment]
> So like this?

yeah this looks good. for the "itoca $name" construction, I've always been treating "toca" as the subject, and the name as the head. I realize it could go either way though but I guess we should pick one.

**Comment by ftyers** [comment]
> > So like this?
> 
> yeah this looks good. for the "itoca $name" construction, I've always been treating "toca" as the subject, and the name as the head. I realize it could go either way though but I guess we should pick one.

I'm going with the predicate is first (because VSO), but I realise that's fairly weak evidence.

**Comment by Lguyogiro** [comment]
That's fine, let's just do that.

**Comment by ftyers** [comment]
Merged in 2aa7492.
